extensions/machine-learning/src/views/models/prediction/predictWizard.ts

```diff
@@ -40,6 +40,7 @@ export class PredictWizard extends ModelViewBase {
 	 * Opens a dialog to manage packages used by notebooks.
 	 */
 	public async open(): Promise<void> {
+		this.modelSourceType = ModelSourceType.RegisteredModels;
```

Comment:
looks good but not sure how this fixes the script issue? was that because of missing t use registered models?

Reply:
ah sorry wrong issue mapped.